src/components/Form.js

```diff
@@ -75,12 +75,18 @@ export default class Form extends Component {
     return null;
   }
 
-  onChange = (formData, options={validate: false}) => {
-    const mustValidate = !this.props.noValidate && (this.props.liveValidate || options.validate);
+  onChange = (formData, newErrorSchema) => {
```

Comment:
Does parameter name `newErrorSchema` make sense here? does `toErrorList(arg)` not evaluate to a "newErrorSchema"?
Also, is this a bit scary to change `onChange`'s parameter API?